frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -137,33 +167,48 @@ const ProjectListView: React.FC = () => {
     setDeleteId(projectNumber);
   };
 
+  const onRowClick = (row: IProject) => {
+    if (mode === PageMode.APPROVAL) {
+      history.push(`/dispose/projects/assess/properties?projectNumber=${row.projectNumber}`);
+    } else {
+      history.push(
+        `/dispose/projects/${row.status.toLowerCase()}?projectNumber=${row.projectNumber}`,
```

Comment:
This comment isn't related to this PR, but to the current route paths.

The problem with using the status as a path parameter it makes it impossible for the user to reference the project in a generic way (i.e. `/dispose/projects/SPP-24234`).  The way we're currently doing this would result in bad conditions if the user goes to the route manually and the project is in a different status.  This is very problematic if for example the project is in a status that shouldn't allow for it.